internal/server/ship/ship.go

```diff
@@ -43,13 +45,14 @@ type Block struct {
 // provide the client with the block list and then send the address of the
 // block that the user selects.
 type Server struct {
-	name           string
-	blocks         []Block
-	shipgateClient api.ShipgateServiceClient
+	name               string
+	blocks             []Block
+	grpcShipgateClient api.ShipgateServiceClient
```

Comment:
Oof, maybe we name this new helper client more specifically to avoid confusion or name mincing like this? What about `ShipListClient` or something

Reply:
Renamed